eclcli/bare/bare_utils.py

```diff
@@ -1,14 +1,17 @@
-#Functions used to do the format work
+# Functions used to do the format work
 import json
```

Comment:
I think that it is necessary to describe like other processing.
```suggestion
try:
    import json
except ImportError:
    import simplejson as json
```